app/javascript/screens/index.js

```diff
@@ -0,0 +1,6 @@
+document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
Lets follow our [JS structure conventions](https://nimblehq.co/compass/development/code-conventions/javascript/) 👍 

Under the screen, we can make a folder with screen name(ex: `dashboard` or any other name)

Then we can define the screen like (pseudocode)
```js

import SomeComponent from 'Components/SomeComponent';


const SELECTORS = {
  screen: 'body.dashboard',
   promotingBanner: '.promoting-banner',
   keuwordList:  '.keyword_list'
  .....
};

class DashboardScreen {
  constructor() {
    this.promotingBanner = document.querySelector(SELECTORS.promotingBanner);
   ...

    this._setup();
  }

  // Private

  _setup() {
    // Here we initialize all the JS component we need for that screen and implement the functionality inside the component
    // ex:  new CSVUploadForm(this.csvUploadForm);, I think your CSV uploading functionality should go in this component
  
  }
}

let isDashboardScreen = document.querySelector(SELECTORS.screen) !== null;

if (isDashboardScreen) {
  new DashboardScreen();
}

```

Reply:
Super interesting! :) I'll dive into that!! Thanks!

Reply:
@junan @ankitkalia1195 
Ok, I have updated & review the JavaScript code to match.

I'm not sure 100% about keeping the form ID as generated (`#new_CSVUploadForm`) or if I should rather use classes to identify the form/file_input. What would you advise here? 

Thanks! :)

Reply:
For selecting DOM elements, we mostly used `class` instead of `id`